projects/1.16/assets/restored-earth/restored_earth/lang/zh_cn.json

```
@@ -0,0 +1,57 @@
+{
+  "entity.restored_earth.muddy_pig": "泥猪",
+  "entity.restored_earth.wooly_cow": "长毛牛",
+  "entity.restored_earth.jumbo_rabbit": "巨兔",
+  "entity.restored_earth.fancy_chicken": "杂色鸡",
+  "entity.restored_earth.rainbow_sheep": " 彩虹羊",
+  "entity.restored_earth.jolly_llama": "欢乐羊驼",
+  "entity.restored_earth.viler_witch": "邪恶女巫",
+  "entity.restored_earth.tropical_slime": "热带史莱姆",
+  "entity.restored_earth.melon_golem": "西瓜傀儡",
+  "entity.restored_earth.moobloom": "哞茛",
```

能说明一下译名的来源吗？没有理解用这个字背后的意图

---

https://minecraft.fandom.com/zh/wiki/Minecraft_Earth:%E5%93%9E%E8%8A%B1
```suggestion
  "entity.restored_earth.moobloom": "哞花",
```
这里的`茛`应该是指毛茛，毕竟都是黄色的（）